### PR TITLE
string localization for task archivefiles

### PR DESCRIPTION
**Pull request by xzhe**
string localization for task archivefiles

### CODE REVIEW

File: Tasks/ArchiveFiles/archivefiles.ts [hunk 1]
```diff
@@ -128,12 +128,7 @@ function tarArchive(archive: string, compression: string, files: string[]) {
 function handleExecResult(execResult, archive) {
     if (execResult.code != tl.TaskResult.Succeeded) {
         tl.debug('execResult: ' + JSON.stringify(execResult));
-        var message = 'Archive creation failed for archive file: ' + archive +
-            '\ncode: ' + execResult.code +
-            '\nstdout: ' + execResult.stdout +
-            '\nstderr: ' + execResult.stderr +
-            '\nerror: ' + execResult.error;
-        failTask(message);
+        failTask(tl.loc('HandleExec', archive, execResult.code, execResult.stdout, execResult.stderr, execResult.error));
```
Comment (yacaovsnc) [inline Tasks/ArchiveFiles/archivefiles.ts]: "HandleExec" is the function, but doesn't convey the meaning of this message.  I suggest a key name such as "ArchiveCreationFailedWithError".

File: Tasks/ArchiveFiles/archivefiles.ts [hunk 2]
```diff
@@ -268,15 +264,15 @@ function doWork() {
                 try {
                     var stats: tl.FsStats = tl.stats(archiveFile);
                     if (stats.isFile()) {
-                        console.log('removing existing archive file before creation: ' + archiveFile);
+                        console.log(tl.loc('RemoveBefore', archiveFile));
                     } else {
-                        failTask('Specified archive file: ' + archiveFile + ' already exists and is not a file.');
+                        failTask(tl.loc('ArchiveFile', archiveFile));
```
Comment (yacaovsnc) [inline Tasks/ArchiveFiles/archivefiles.ts]: Again, this key is also too generic. Consider something like "ArchiveFileExistsButNotAFile"?

**Comment by yacaovsnc** [comment]
You should also compile and checkin the "task.loc.json" and the resource loc file.  Otherwise every time other people compile they will see dirty files.

**Comment by xzhe** [comment]
Done! Thanks for the comments!

**Pull request merged**